storey/steps/sample.py

```diff
@@ -22,13 +23,22 @@ class SampleWindow(Flow):
     Available options:
         2.1) True - The last event seen will be emitted downstream.
         2.2) False - The last event seen will NOT be emitted downstream.
+    :param count_per_key: Whether to sample up to window size on per key basis. False samples on all events.
```

Comment:
`count_per_key` is redundant. It's always used as `key and count_per_key`. Can just use `key`.